src/sass/components/_box.scss

```diff
@@ -2,15 +2,52 @@
     display: flex;
     flex-direction: column;
     justify-content: space-between;
+
+    &--about {
+        @media (min-width: $breakpoint-md) {
+            display: flex;
+            flex-direction: row;
+            justify-content: space-between;
+        }
+    }
+
+    &--journey {
+        @media (min-width: $breakpoint-md) {
+            display: flex;
+            flex-direction: row;
+            flex-wrap: wrap;
+        }
+    }
+
+    &--faq {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+    }
 }
 
 .box {
+    $journey-border-color: #e2e2e2;
+    $journey-border: 0.5px solid $journey-border-color;
+    $journey-width-breakpoint-md: 40vw;
```

Comment:
breakpoints should be global - you should avoid creating different breakpoints

Reply:
So, it's not okay to put different values for different breakpoints. 
Better should be global like: 
$width-xs: 10vw;
$width-sm: 20vw;
$width-md: 30vw;
$width-lg: 40vw;

Reply:
yep